File: 240418/누적합이 200/cumulative-sum-200.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.
    int num, array[100] = { 0 };

    scanf("%d", &num);

   int i;
    for(i = 0; i < num; i++) {
        scanf("%d", &array[i]);
    }

    int j, sum = 0;
    for(j = 0; j < num; j++) {
        sum += array[j];
        if(sum > 200) {
            printf("%d\n", sum);

            double average = (double) sum / (j + 1); 
            printf("%.1lf\n", average);
            
            return 0;
        }
    }

    return 0;
}

File: 240418/누적합이 200/cumulative-sum-200.kt
fun main() {
    val num = readLine()?.toIntOrNull() ?: return
    val input = readLine()?.split(" ")?.map { it.toIntOrNull() } ?: return

    val array = IntArray(num) { 0 }
    var sum = 0

    for (i in 0 until num) {
        array[i] = input[i] ?: return
        sum += array[i] ?: return

        if (sum > 200) {
            println(sum)

            val average = sum.toDouble() / (i + 1)
            println("%.1f".format(average))
            
            return
        }
    }
}